### PR TITLE
try nicer loading

### PR DESCRIPTION
**Pull request by grosser**

### CODE REVIEW

**Comment by grosser** [comment]
FYI @maximeg this is a bit nicer and faster :)

**Comment by maximeg** [comment]
Good to know ;)

File: lib/gettext_i18n_rails/base_parser.rb [hunk 1]
```diff
@@ -23,12 +23,7 @@ def self.load_library
       return true if @library_loaded
 
       loaded = libraries.detect do |library|
-        begin
-          require library
-          true
-        rescue LoadError
-          false
-        end
+        require library if Gem::Specification.find_all_by_name(library).any?
```
Comment (dgilperez) [inline lib/gettext_i18n_rails/base_parser.rb]: I believe this is not producing the same output: 

`require library` can return true (if loading OK), false (if it was already loaded) or a `LoadError`. Thus, when the library was already loaded, we're thinking the library is not present.

I'm preparing a patch.

Comment (grosser) [inline lib/gettext_i18n_rails/base_parser.rb]: ah good catch! ... did not think of the return value :/

**Pull request merged**